src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Startup.cs

```diff
@@ -7,6 +7,7 @@
 using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using GrpcService_CSharp.Services;
```

Comment:
Alphabetical order usings

Reply:
Move above Microsoft usings and below system

Reply:
Doesn't the order end up depending on the user's project name?

Reply:
Hmm, good point.